homeassistant/components/demo/geo_location.py

```diff
@@ -59,26 +62,24 @@ def _generate_random_event(self):
         home_longitude = self._hass.config.longitude
 
         # Approx. 111km per degree (north-south).
-        radius_in_degrees = random.random() * MAX_RADIUS_IN_KM / AVG_KM_PER_DEGREE
+        radius_in_degrees = random.random(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Standard pseudo-random generators are not suitable for security/cryptographic purposes.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4636028)